pizza-recipes.md

```diff
@@ -174,3 +174,78 @@ Embrace this blend of flavors as you forgo store-brought sauce for this nuturiou
 6. Cook for 30 mins, adjust water if thick.
 
 7. Let it cool. Your pizza sauce is ready to use.
+## Nigeria pizza
+![Nigeria-suya-pizza](images/nigeria-suya-pizza.jpg)
+
+The first experience with this snack might not be interesting. It may taste strange for your first time especially for those with Nigeria taste bud.lol! But not to worry with time you will get used to taste and will start running back for more of it.
```

Comment:
[proofreading] This is a very humorous way to start a recipe. 😁 I just made some grammar edits to make that clear.
```suggestion
The first experience with this snack might taste strange, especially for those without Nigerian tastebuds lol! 😁 But not to worry, with time you will get used to the taste and will start running back for more! 😁
```